<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>svntogit/packages.git - Git clone of the 'packages' repository
</title>
<meta name='generator' content='cgit v0.12'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='https://git.archlinux.org/svntogit/packages.git/atom/trunk/0001-fix-test-with-freetype2-2.7.1.patch?h=packages/fontconfig' type='application/atom+xml'/>
<link rel='vcs-git' href='https://git.archlinux.org/svntogit/packages.git' title='svntogit/packages.git Git repository'/>
<link rel='vcs-git' href='git://git.archlinux.org/svntogit/packages.git' title='svntogit/packages.git Git repository'/>
<link rel='vcs-git' href='ssh://git.archlinux.org/srv/git/svntogit/packages.git' title='svntogit/packages.git Git repository'/>
</head>
<body>
	<div id="archnavbar"><!-- Arch Linux global navigation bar -->
		<div id="archnavbarlogo">
			<p><a href="http://www.archlinux.org/" title="Arch news, packages, projects and more"></a></p>
		</div>
		<div id="archnavbarmenu">
			<ul id="archnavbarlist">
				<li id="anb-home"><a href="http://www.archlinux.org/" title="Arch news, packages, projects and more">Home</a></li>
				<li id="anb-packages"><a href="http://www.archlinux.org/packages/" title="Arch Package Database">Packages</a></li>
				<li id="anb-forums"><a href="https://bbs.archlinux.org/" title="Community forums">Forums</a></li>
				<li id="anb-wiki"><a href="https://wiki.archlinux.org/" title="Community documentation">Wiki</a></li>
				<li id="anb-bugs"><a href="https://bugs.archlinux.org/" title="Report and follow bugs">Bugs</a></li>
				<li id="anb-aur"><a href="https://aur.archlinux.org/" title="Arch Linux User Repository">AUR</a></li>
				<li id="anb-download"><a href="http://www.archlinux.org/download/" title="Get Arch Linux">Download</a></li>
			</ul>
		</div>
	</div><!-- #archnavbar -->
<div id='cgit'><table id='header'>
<tr>
<td class='main'><a href='/'>index</a> : <a title='svntogit/packages.git' href='/svntogit/packages.git/'>svntogit/packages.git</a></td></tr>
<tr><td class='sub'>Git clone of the 'packages' repository
</td><td class='sub right'></td></tr></table>
<table class='tabs'><tr><td>
<a href='/svntogit/packages.git/?h=packages/fontconfig'>summary</a><a href='/svntogit/packages.git/refs/?h=packages/fontconfig'>refs</a><a href='/svntogit/packages.git/log/trunk/0001-fix-test-with-freetype2-2.7.1.patch?h=packages/fontconfig'>log</a><a class='active' href='/svntogit/packages.git/tree/trunk/0001-fix-test-with-freetype2-2.7.1.patch?h=packages/fontconfig'>tree</a><a href='/svntogit/packages.git/commit/trunk/0001-fix-test-with-freetype2-2.7.1.patch?h=packages/fontconfig'>commit</a><a href='/svntogit/packages.git/diff/trunk/0001-fix-test-with-freetype2-2.7.1.patch?h=packages/fontconfig'>diff</a><a href='/svntogit/packages.git/stats/trunk/0001-fix-test-with-freetype2-2.7.1.patch?h=packages/fontconfig'>stats</a></td><td class='form'><form class='right' method='get' action='/svntogit/packages.git/log/trunk/0001-fix-test-with-freetype2-2.7.1.patch'>
<input type='hidden' name='h' value='packages/fontconfig'/><select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/svntogit/packages.git/tree/?h=packages/fontconfig'>root</a>/<a href='/svntogit/packages.git/tree/trunk?h=packages/fontconfig'>trunk</a>/<a href='/svntogit/packages.git/tree/trunk/0001-fix-test-with-freetype2-2.7.1.patch?h=packages/fontconfig'>0001-fix-test-with-freetype2-2.7.1.patch</a></div><div class='content'>blob: d834b707a1a0ea8df49d0eef9e1bbe14c1a9e5d7 (<a href='/svntogit/packages.git/plain/trunk/0001-fix-test-with-freetype2-2.7.1.patch?h=packages/fontconfig'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
<a id='n34' href='#n34'>34</a>
<a id='n35' href='#n35'>35</a>
<a id='n36' href='#n36'>36</a>
<a id='n37' href='#n37'>37</a>
<a id='n38' href='#n38'>38</a>
<a id='n39' href='#n39'>39</a>
<a id='n40' href='#n40'>40</a>
<a id='n41' href='#n41'>41</a>
<a id='n42' href='#n42'>42</a>
</pre></td>
<td class='lines'><pre><code>From 165f0f012f8d4bebb12d188c1c2a369d233d586e Mon Sep 17 00:00:00 2001
From: Christian Hesse &lt;mail&#64;eworm.de&gt;
Date: Tue, 10 Jan 2017 10:01:54 +0100
Subject: [PATCH 1/1] fix test with freetype2 2.7.1

Upstream freetype2 applied a patch from openSuSE [0]:

[pcf] Enrich family name with foundry name and glyph width info.
<span class="hl num">3576487a</span>dd2f0e9691e8c43f8b0cd8e61aebf14f

This prepends the foundry name plus a space to the family name. We needs
change the font name in expexted test output.

[0] http://git.savannah.gnu.org/cgit/freetype/freetype2.git/commit/?id=3576487add2f0e9691e8c43f8b0cd8e61aebf14f

Signed-off-by: Christian Hesse &lt;mail&#64;eworm.de&gt;
<span class="hl kwb">---</span>
 test/out.expected | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/test/out.expected b/test/out.expected
index 39634c5..f3a8856 100644
<span class="hl kwb">--- a/test/out.expected</span>
<span class="hl kwa">+++ b/test/out.expected</span>
<span class="hl kwd">&#64;&#64; -1,8 +1,8 &#64;&#64;</span>
<span class="hl kwb">-Fixed:pixelsize=16</span>
<span class="hl kwb">-Fixed:pixelsize=6</span>
<span class="hl kwa">+Misc Fixed:pixelsize=6</span>
<span class="hl kwa">+Sony Fixed:pixelsize=16</span>
 =
<span class="hl kwb">-Fixed:pixelsize=16</span>
<span class="hl kwb">-Fixed:pixelsize=6</span>
<span class="hl kwa">+Misc Fixed:pixelsize=6</span>
<span class="hl kwa">+Sony Fixed:pixelsize=16</span>
 =
<span class="hl kwb">-Fixed:pixelsize=16</span>
<span class="hl kwb">-Fixed:pixelsize=6</span>
<span class="hl kwa">+Misc Fixed:pixelsize=6</span>
<span class="hl kwa">+Sony Fixed:pixelsize=16</span>
<span class="hl kwb">-- </span>
2.11.0

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class="foot" style="padding-left:1em;padding-right:1em;">
<p>Copyright &copy; 2002-2016 <a href="mailto:jvinet@zeroflux.org"
title="contact Judd Vinet">Judd Vinet</a> and <a href="mailto:aaron@archlinux.org"
title="contact Aaron Griffin">Aaron Griffin</a>. The Arch Linux name and logo
are recognized trademarks. Some rights reserved. The registered trademark
Linux&reg; is used pursuant to a sublicense from LMI, the exclusive licensee
of Linus Torvalds, owner of the mark on a world-wide basis.</p>
</div>
</div> <!-- id=cgit -->
</body>
</html>
